client/components/modals/SelectKeywords.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useEffect } from 'react';
+
+import { keywordsSamples } from '../../constants/keywords-sample';
+
+import {
+  Box,
+  Heading,
+  Input,
+  InputGroup,
+  InputLeftAddon,
+  Container,
+  Text,
+  Stack,
+  SimpleGrid,
+  Tag,
+  TagLabel,
+  TagCloseButton,
+  TagRightIcon,
+  ButtonGroup,
+} from '@chakra-ui/react';
+
+import {
+  ButtonBlue,
+  ButtonOrange,
+} from '../../styles/ui-components/Chakra-Button';
+
+import { AddIcon, SearchIcon } from '@chakra-ui/icons';
+
+type KeywordProps = {
+  keywordsData: any;
+  keywordsDataHandler: (keywordData: React.FormEvent) => void;
+  closeKeywordModal: () => void;
+};
+
+export default function KeywordSelect({
+  keywordsDataHandler,
+  closeKeywordModal,
+  keywordsData,
+}: KeywordProps): any {
+  let keywords = keywordsSamples;
+
+  const [searchKeywords, setSearchKeywords] = useState('');
+  const [selectedKeywords, setSelectedKeywords] = useState<any>([]);
+  const [keywordsActive, setKeywordsActive] = useState(false);
+
+  useEffect(() => {
+    if (keywordsData && selectedKeywords.length === 0) {
+      Object.values(keywordsData).forEach((keyword) => {
+        selectKeyword((keyword as any).keyword);
+      });
+    }
+  }, [selectedKeywords, keywordsData]);
+
+  const selectKeyword = (e: string) => {
+    //Limits the number of selected keywords to 10
+    const isInArray = selectedKeywords.find(
+      (element: any) => element.keyword === e
+    );
+    if (isInArray || selectedKeywords.length >= 10) {
+      return;
+    }
```

Comment:
Shouldn't this just be?
`if (isInArray || selectedKeywords.length > 10) { return; }`

Having it `=== 10` is desired, no?

Reply:
Assuming my thoughts are incorrect then?